### PR TITLE
Update `CHANGELOG.md` with  #9154 breaking changes.

### PR DESCRIPTION
**Pull request by EstherPerelman**
Signed-off-by: Esther Perelman <esther.perelman@sap.com>

#### What it does
[#9154 comment](https://github.com/eclipse-theia/theia/pull/9154#issuecomment-879682898)

#### Review checklist

- [x] as an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

### CODE REVIEW

**Comment by EstherPerelman** [comment]
@tsmaeder is it okay?

File: CHANGELOG.md [hunk 1]
```diff
@@ -49,6 +49,7 @@
 
 <a name="breaking_changes_1.15.0">[Breaking Changes:](#breaking_changes_1.15.0)</a>
 
+- [monaco] upgraded to monaco 0.23.0 including replacement of `quickOpen` API (0.20.x) with `quickInput` API (0.23.x) [#9154](https://github.com/eclipse-theia/theia/pull/9154)
```
Comment (msujew) [inline CHANGELOG.md:52]: As #9154 has been merged after the release of 1.15.0, I'd argue that this should be moved to the breaking changes section of 1.16.0.

Also `upgraded to monaco 0.23.0` will probably part of the normal changelog. Therefore, the breaking changes should only include `replacement of 'quickOpen' API (monaco 0.20.x) with 'quickInput' API (monaco 0.23.x)`

Comment (EstherPerelman) [reply CHANGELOG.md]: Right! updated...

**Review by vince-fugnitto — APPROVED** [review]
The changes look good to me 👍 I generally do a pass of the changelog during the month for the release if anything was ever missed :)

**Pull request merged**